resources/definitions/lulzbot_taz_pro_dual.def.json

```diff
@@ -0,0 +1,34 @@
+{
+   "version": 2,
+   "name": "Lulzbot TAZ Pro Dual Extruder",
+   "inherits": "lulzbot_taz_pro_common",
+   "metadata": {
+        "author": "Tom Callaway <spot@fedoraproject.org>",
+        "manufacturer": "Fargo Additive Manufacturing Equipment 3D, LLC",
+        "visible": true,
+        "file_formats": "text/x-gcode",
+        "platform": "lulzbot_taz_pro_bed.stl",
+        "platform_offset": [0, -25, 0],
```

Comment:
The offset should be around -10, or else the platform is lower than the buildplate in Cura
```suggestion
        "platform_offset": [0, -10, 0],
```

"platform_offset": [0, -25, 0]:
![image](https://user-images.githubusercontent.com/19388042/110145267-8a80bc80-7dd9-11eb-94b9-d58bace8edb4.png)

"platform_offset": [0, -10, 0]

![image](https://user-images.githubusercontent.com/19388042/110145508-ce73c180-7dd9-11eb-94b5-44f3240c8556.png)